tests/integration_tests/bank/case.go

```diff
@@ -611,32 +616,60 @@ func run(
 	}
 }
 
-func getDownStreamSyncedEndTs(ctx context.Context, db *sql.DB, tableName string) (result string, err error) {
+func getDownStreamSyncedEndTs(ctx context.Context, db *sql.DB, tidbAPIEndpoint, tableName string) (result uint64, err error) {
 	for {
 		select {
 		case <-ctx.Done():
 			log.Error("get downstream sync end ts failed due to timeout", zap.String("table", tableName), zap.Error(ctx.Err()))
-			return "", ctx.Err()
+			return 0, ctx.Err()
 		case <-time.After(2 * time.Second):
-			result, ok := tryGetEndTs(db, tableName)
+			result, ok := tryGetEndTs(db, tidbAPIEndpoint, tableName)
 			if ok {
 				return result, nil
 			}
 		}
 	}
 }
 
-func tryGetEndTs(db *sql.DB, tableName string) (result string, ok bool) {
-	query := "SELECT END_TIME FROM information_schema.ddl_jobs WHERE table_name = ?"
-	log.Info("try get end ts", zap.String("query", query))
-	var endTime string
+func tryGetEndTs(db *sql.DB, tidbAPIEndpoint, tableName string) (result uint64, ok bool) {
+	query := "SELECT JOB_ID FROM information_schema.ddl_jobs WHERE table_name = ?"
```

Comment:
```suggestion
	// Note: We should not use `END_TS` in the table, because it is encoded in
	// the format `2023-03-16 18:12:51`, it's not precise enough.
	query := "SELECT JOB_ID FROM information_schema.ddl_jobs WHERE table_name = ?"
```